chart/templates/services/admin/_container.tpl

```diff
@@ -12,6 +12,10 @@
   {{ include "envCommon" . | nindent 2 }}
   {{ include "envLog" . | nindent 2 }}
   {{ include "envMetrics" . | nindent 2 }}
+  {{ include "envParquetMetadata" . | nindent 2 }}
+  {{ include "envWorker" . | nindent 2 }}
```

Comment:
Not sure if we should include all the worker configs here, maybe just the ones that are needed? (I think it is DUCKDB_INDEX_CACHE_DIRECTORY and DESCRIPTIVE_STATISTICS_CACHE_DIRECTORY)

Reply:
Yes, we might be more precise. I'll change it, also: it will be clearer in the admin template